package.json

```diff
@@ -1,110 +1,119 @@
-    "prop-types": "^15.5.10",
-    "react": "^16.8.6",
-    "react-addons-css-transition-group": "^15.6.2",
-    "react-addons-shallow-compare": "^15.5.2",
-    "react-bootstrap": "^0.32.1",
-    "react-bootstrap-slider": "^2.1.5",
-    "react-dom": "^16.8.6",
-    "react-intl": "^2.4.0",
-    "react-scripts": "3.0.1",
-    "rtl-detect": "^1.0.0",
-    "string-punctuation-tokenizer": "^2.0.0-beta.2",
-    "sweetalert": "^2.1.0",
-    "sweetalert-react": "^0.4.11",
-    "transform-runtime": "0.0.0"
-  },
-  "build": {
-    "extends": "react-cra",
-    "appId": "oa.autographa.lite",
-    "files": [
-      "build/**/*.*",
-      "public/**/*.*",
-      "node_modules/**/*",
-      "src/**/*.*",
-      "patterns.prop"
-    ],
-    "directories": {
-      "buildResources": "resources"
-    },
-    "mac": {
-      "category": "public.app-category.Utilities",
-      "target": "dmg",
-      "icon": "resources/icon.icns",
-      "type": "distribution"
-    },
-    "linux": {
-      "target": [
-        "deb"
-      ],
-      "icon": "resources/icon.png"
-    },
-    "publish": null
-  },
-  "keywords": [
-    "Desktop",
-    "Bible Editor",
-    "USFM",
-    "USFM Editor",
-    "React.js",
-    "Translation",
-    "Open Source"
-  ],
-  "scripts": {
-    "start": "concurrently \"npm run react-start\" \"wait-on http://localhost:3000/ && cross-env NODE_ENV=development npm run electron-start\"",
-    "build": "npm run build:electron",
-    "electron-start": "electron .",
-    "electron-build": "electron-packager ./build --out=dist",
-    "react-start": "craco start",
-    "react-build": "craco build",
-    "test": "craco test",
-    "eject": "react-scripts eject",
-    "build:electron": "craco build --config craco.config.js main=build/electron.js",
-    "preelectron-pack": "npm run build",
-    "electron-pack": "electron-builder build"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ],
-  "devDependencies": {
-    "@rescripts/cli": "0.0.10",
-    "@rescripts/rescript-env": "0.0.10",
-    "babel-plugin-react-intl": "^3.0.1",
-    "concurrently": "^4.1.0",
-    "cross-env": "^5.2.0",
-    "electron": "^6.0.9",
-    "electron-builder": "^21.2.0",
-    "electron-packager": "^13.1.1",
-    "electron-rebuild": "^1.8.4",
-    "wait-on": "^3.2.0"
-  }
+	"name": "autographa-lite",
+	"description": "A Bible translation editor for everyone.",
+	"version": "1.3.4",
+	"main": "public/electron.js",
+	"author": "Friends of Agape <mathew.joel@protonmail.com>",
+	"license": "MIT",
+	"homepage": ".",
+	"dependencies": {
+		"@craco/craco": "^3.6.0",
+		"@material-ui/core": "^4.2.0",
+		"@material-ui/icons": "^4.2.1",
+		"chai-as-promised": "^7.1.1",
+		"customize-cra": "^0.2.12",
+		"diff-match-patch": "^1.0.0",
+		"electron-squirrel-startup": "^1.0.0",
+		"electron-log": "^2.2.7",
+		"fs-jetpack": "^1.0.0",
+		"jquery": "^3.3.1",
```

Comment:
Maybe not in this PR but we should see if we can get rid of `jquery` and possibly other such obsolete dependencies.

Reply:
yes I will check and clear those

Reply:
jquery is a kind of peer dependency for bootstrap-slider. So we can remove until bootstrap slider exists